ranger/ext/keybinding_parser.py

```diff
@@ -53,7 +53,7 @@ def special_keys_init():
     for key, val in tuple(special_keys.items()):
         special_keys['a-' + key] = (ALT_KEY, val)
 
-    for char in 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789_!{}':
+    for char in 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789_!{}[],./':
```

Comment:
In vim, `<C-F>` stands for `<CTRL>+f`. Users are not allowed to bind `<CTRL>` or `<ALT>` with an uppercase letter. Just remove all uppercase letters here and forbid users bind a combination key with an uppercase letter also fixes this.

But in [`examples/plugin_pmount.py#L16-#L18`](https://github.com/ranger/ranger/blob/master/examples/plugin_pmount.py#L16-#L18), the example binds an uppercase letter with `<alt>`:

```python
MOUNT_KEY = '<alt>m'
UMOUNT_KEY = '<alt>M'
LIST_MOUNTS_KEY = '<alt>N'
```

Reply:
New special key of `<A-/>` is added here for fd integration.
https://github.com/ranger/ranger/wiki/Custom-Commands#search-with-fd